packages/gatsby-theme-store/src/gatsby-ssr.tsx

```diff
@@ -1,7 +1,18 @@
+import { UIProvider } from '@vtex/store-sdk'
 import React from 'react'
-import type { RenderBodyArgs } from 'gatsby'
+import type { RenderBodyArgs, WrapRootElementNodeArgs } from 'gatsby'
 
 import { handleUserBackOnline } from './sdk/offline'
+import { Provider as RegionProvider } from './sdk/region/Provider'
+import { Provider as ToastProvider } from './sdk/toast/Provider'
+
+export const wrapRootElement = ({ element }: WrapRootElementNodeArgs) => (
+  <ToastProvider>
+    <RegionProvider>
+      <UIProvider>{element}</UIProvider>
+    </RegionProvider>
+  </ToastProvider>
```

Comment:
The `UIProvider` should be the first provider, because `ToastProvider` can use then.
```suggestion
  <UIProvider>
    <RegionProvider>
      <ToastProvider>{element}</ToastProvider>
    </RegionProvider>
  </UIProvider>
```